addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -266,10 +264,10 @@ odoo.define('website_slides.quiz', function (require) {
     });
 
     sAnimations.registry.websiteSlidesQuizNoFullscreen = sAnimations.Class.extend({
-        selector: '.o_wslides_js_lesson_quiz',
+        selector: '.o_wslides_lesson_main', // selector of completed page, as we need slide content and aside content table
```

Comment:
completed or complete? Not a big deal but a little confusing.

Reply:
indeed !